Make library minor number 100 + Netpbm minor number so it is higher than previous ones in Major 11
https://sourceforge.net/p/netpbm/code/4467/
--- lib/Makefile.orig	2022-11-09 12:16:24.000000000 -0600
+++ lib/Makefile	2022-11-16 15:43:11.000000000 -0600
@@ -100,7 +100,7 @@
 	$(CC) -c $(INCLUDES) $(CFLAGS_ALL) -o $@ $<
 
 MAJ = 11
-MIN = $(NETPBM_MINOR_RELEASE)
+$(eval MIN:=$(shell echo $$((100+$(NETPBM_MINOR_RELEASE)))))
 
 SONAME = libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ)
 
